packages/genesys-spark-components/src/components/stable/gux-icon/example.html

```diff
@@ -5,6 +5,7 @@ <h1>gux-icon</h1>
     <gux-tab tab-id="1-1">Usage</gux-tab>
     <gux-tab tab-id="1-2">Current icons</gux-tab>
     <gux-tab tab-id="1-3">Legacy icons</gux-tab>
+    <gux-tab tab-id="1-4">Restricted icons</gux-tab>
```

Comment:
I also considered making the 3rd tab 'Legacy & Restricted' as they are both kind of discouraged. Let me know if you'd prefer that